FrontEnd/public/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
     <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
-    <link rel="manifest" href="/site.webmanifest" />
+<!--    <link rel="manifest" href="/site.webmanifest" />-->
```

Comment:
Are we going to implement PWA? If no we can remove it

Reply:
Thank you for feedback. Done.